File: p5.c
#include <stdlib.h>
#include <stdio.h>

//Author : Mackenzie Park-McInnes
//Title : P5
//Purpose : Solve 'projecteuler.net' archived problem 5
//Compilation : gcc -o p5 p5.c

int main() {
	int sm = 1;
	int i;
	int td = 1;

	while (1) {
		for(i=1; i <= 20; i++) {
			if (sm % i != 0) {
				td = 0;
				break;
			}
		}
		
		if (td == 1) {
			printf("%d", sm);
			break;
		} else {
			td = 1;
			sm++;
		}
	}
}

File: p5.rb
#Author : Mackenzie Park-McInnes
#Title : P5
#Purpose : Solve 'projecteuler.net' archived problem 5

td = 1
i = 1

while 1 do
	for j in 1..20
		if i % j != 0
			td = 0
			break
		end
	end
	if td == 1
		puts "\n", i
		break
	else
		td = 1
		i += 1
	end
end